modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc

```diff
@@ -5,6 +5,13 @@ Base the file name and the ID on the module title. For example:
 * Title: = My concept module A
 ////
 
+////
+Indicate the module type in one of the following
+ways:
+Add the prefix con- or con_ to the file name.
+Add `:_module-type: CONCEPT` before the module ID. 
```

Comment:
This being a comment (i.e. it won't be rendered anywhere), I don't see why the code should be enclosed in backticks.

```suggestion
Add the following attribute before the module ID:
:_module-type: CONCEPT
```